.github/filters.yaml

```diff
@@ -1,11 +1,15 @@
 backend:
+  - '.github/actions/yarn-nm-install/*.yml'
```

Comment:
Shouldn't exactly be part of this P/R... but asked in https://github.com/strapi/strapi/pull/16581#issuecomment-1536662596.

Added the yarn-nm-install folder too. Why ? I'll need it for second iteration. But don't worry it's part of a plan: https://github.com/strapi/strapi/pull/16581#issuecomment-1537120493